### ISSUE TITLE
OIT not working correctly in Firefox

### ISSUE THREAD

**Issue by lilleyse** [AnalyticalGraphicsInc/cesium#4658]
Blending vs. OIT
<img width="274" alt="good" src="https://cloud.githubusercontent.com/assets/915398/20361029/107b5c5c-ac04-11e6-8329-49789afe81a2.PNG">
<img width="254" alt="bad" src="https://cloud.githubusercontent.com/assets/915398/20361028/107570a8-ac04-11e6-9991-42bad7455d32.PNG">

I have only noticed this on Firefox so far, but will check Chrome on a different computer later.

**Comment by pjcozzi** [comment]
Is this new in Firefox?  It may be a bug in Firefox or a change in the Firefox WebGL implementation that exposed a bug in OIT.

**Comment by lilleyse** [comment]
Yeah this is new, at most about a month.

**Comment by hpinkos** [comment]
Also reported here: #4718

**Comment by jroetman** [comment]
A temporary workaround in my case was to update the 
webgl.disable-extensions = true

Although, I mucked around with other options too, and it seems that certain combinations break everything, so here's my working setup of webgl config properties:

![image](https://cloud.githubusercontent.com/assets/5098704/20992163/429017f2-bc97-11e6-981d-de2eed4e0788.png)

**Comment by jmgomezpoveda** [comment]
Setting `webgl.disable-extensions = true` in my browser also fixed the issue. Though it is not really a workaround since we cannot ask users to change it in their browsers.

**Comment by pjcozzi** [comment]
As a workaround, try setting `orderIndependentTranslucency : false` when constructing the Cesium `Viewer` widget.

**Comment by jmgomezpoveda** [comment]
@pjcozzi that workaround works great on my system.

**Comment by hpinkos** [comment]
also reported here: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/Kd3gJciiCkQ

**Comment by mramato** [comment]
We should probably look into this sooner, rather than later.

**Comment by mramato** [comment]
I noticed Firefox is spitting out tons of the below errors (and I believe this is a new development).  Maybe there's a bug in some new code they added.

```
Error: WebGL: texImage2D: Incurred CPU row conversion, which is slow.  Texture.js:147:17
Error: WebGL: texSubImage2D: Incurred CPU row conversion, which is slow.
```

**Comment by mramato** [comment]
Last update:
1. The above errors occur even when OIT is off
2. The rendering problem still happens even with ANGLE off
3. Possibly related to #3068

**Comment by pjcozzi** [comment]
@jroetman @jmgomezpoveda this should be fixed now via #4762.  It will be in Cesium 1.29 on January 2.

**Comment by jmgomezpoveda** [comment]
Yeah this works fine now in Firefox 50. Thanks!
